src/features/classification/security-controls/__tests__/utils.test.js

```diff
@@ -184,6 +247,29 @@ describe('features/classification/security-controls/utils', () => {
             },
         );
 
+        test.each([WHITELIST, BLACKLIST])(
+            'should include correct message when integration download is restricted by %s and integrations are less than maxAppCount and shouldDisplayAppsAsIntegrations is true',
+            listType => {
+                accessPolicy = {
+                    app: {
+                        accessLevel: listType,
+                        apps: [{ displayText: 'a' }, { displayText: 'b' }, { displayText: 'c' }],
+                    },
+                };
+                const expectedMessage = integrationRestrictionsMessageMap[listType][WITH_APP_LIST];
+
+                expect(expectedMessage).toBeTruthy();
+                expect(getFullSecurityControlsMessages(accessPolicy, 3, true)).toEqual([
+                    {
+                        message: {
+                            ...expectedMessage,
+                            values: { appNames: 'a, b, c' },
+                        },
+                    },
+                ]);
+            },
+        );
+
```

Comment:
I think we need assert against the hardcoded message also, the test is essentially check if the input is provide the correct messages so using the variable here might not be useful because someone could still change the message and these unit tests will still pass same with the test below.

if we do not care about the message in these unit tests and we are just testing the function itself, then maybe updating the test name to reflect that